plugins/LadspaEffect/calf/CMakeLists.txt

```diff
@@ -38,7 +38,10 @@ SET(INLINE_FLAGS "")
 IF("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
 	SET(INLINE_FLAGS -finline-functions-called-once -finline-limit=80)
 ENDIF()
-target_compile_options(veal PRIVATE -fexceptions -O2 -finline-functions ${INLINE_FLAGS})
+
+IF(NOT MSVC)
+	target_compile_options(veal PRIVATE -fexceptions -O2 -finline-functions ${INLINE_FLAGS})
+endif()
```

Comment:
It looks like MSVC builds don't have the `/O2` flag to turn on optimizations. 

There may also be MSVC flags corresponding to the GCC inlining flags, but they do not seem to be as important.

Reply:
Sorry for the accidental resolve.

The reason why i disabled those conditionally is because some of these flags is invalid on msvc. I need to find the equivalent flags and do some cmake tweaks but will get to it when i get the time.

Reply:
`/O2` is MSVC's equivalent of GCC's `-O2`. And [according to Microsoft's documentation](https://learn.microsoft.com/en-us/cpp/build/reference/ob-inline-function-expansion?view=msvc-170), the `/O2` flag will automatically use `/Ob2` for inlining functions.

This should be fine:
```cmake
if(MSVC)
	target_compile_options(veal PRIVATE /O2)
else()
	target_compile_options(veal PRIVATE -fexceptions -O2 -finline-functions ${INLINE_FLAGS})
endif()
```

Reply:
Actually, looking at [this article](https://learn.microsoft.com/en-us/cpp/build/reference/o1-o2-minimize-size-maximize-speed?view=msvc-170), `/O2` is the default for Release builds, so maybe no changes are needed.

Reply:
I'm resolving this for now.